src/test/java/api/support/fixtures/AgeToLostFixture.java

```diff
@@ -39,24 +44,31 @@ public AgeToLostFixture(ItemsFixture itemsFixture, UsersFixture usersFixture,
   }
 
   public AgeToLostResult createAgedToLostLoan() {
-    policiesActivation.useLostItemPolicy(
-      lostItemFeePoliciesFixture.ageToLostAfterOneMinute().getId());
+    return createAgedToLostLoan(PoliciesToActivate.builder()
+    .lostItemPolicy(lostItemFeePoliciesFixture.ageToLostAfterOneMinute()));
+  }
+
+  public AgeToLostResult createAgedToLostLoan(PoliciesToActivate.PoliciesToActivateBuilder policiesToUse) {
+    policiesActivation.use(policiesToUse);
 
     val user = usersFixture.james();
     val item = itemsFixture.basedUponNod(ItemBuilder::withRandomBarcode);
-    val loan = checkOutFixture
-      .checkOutByBarcode(item, user, getLoanOverdueDate().minusMinutes(2));
+    val loan = checkOutFixture.checkOutByBarcode(item, user);
 
-    timedTaskClient.start(scheduledAgeToLostUrl(), 204, "scheduled-age-to-lost");
+    try {
+      // Go to the future
+      getClockManager().setClock(fixed(ofEpochMilli(now().plusMonths(6).getMillis()), ZoneOffset.UTC));
+      timedTaskClient.start(scheduledAgeToLostUrl(), 204, "scheduled-age-to-lost");
+    } finally {
+      getClockManager().setDefaultClock();
+    }
```

Comment:
What is this code intended to do?

I would prefer that there wasn't discretionary code within a test. If the intention is to reset the clock whether the test succeeds or fails, I suggest adding before and after methods to the test class to do so.

Reply:
There should be already an after method in `APITests` that restores the clocks. So I'll just rely on it and remove the finally block.

Reply:
the try/finally is removed now.